File: Python/pangram.py
class Solution:
    def checkIfPangram(self, sentence):
        uniqueSet = set()
        for ch in sentence:
            currChar = ch.lower()
            if currChar.isalpha():
                uniqueSet.add(currChar)

        if len(uniqueSet) != 26:
            return False

        return True

File: Swift/Meta/Easy/Pangram.swift
//5ms
class Solution {
    func checkIfPangram(_ sentence: String) -> Bool {
        var uniqueSet = Set<Character>()

        for char in sentence {
            //Leetcode says sentence only contains lowercase letters
            guard (char.isLetter || char.isNumber) else {
                continue
            }
            uniqueSet.insert(char)
        }

        return uniqueSet.count == 26
    }
}
